src/commands/deprovision/server/deprovisionserver.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"phoenixnap.com/pnapctl/common/client/bmcapi"
+	"phoenixnap.com/pnapctl/common/models/bmcapimodels/servermodels"
+	"phoenixnap.com/pnapctl/common/utils"
+)
+
+// Filename is the filename from which to retrieve a complex object
+var Filename string
+
+var commandName = "deprovision server"
+
+// DeprovisionServerCmd
+var DeprovisionServerCmd = &cobra.Command{
+	Use:          "server SERVER_ID",
+	Short:        "Deprovision a server.",
+	Args:         cobra.ExactArgs(1),
+	SilenceUsage: true,
+	Long: `Deprovision a server.
+
+Requires a file (yaml or json) containing the information needed to deprovision a server.`,
+	Example: `# Deprovision a server as per serverdeprovision.yaml
+pnapctl deprovision server <SERVER_ID> --filename <FILE_PATH>
```

Comment:
If we're specifying `as per serverdeprovision.yaml`, should we replace `<FILE_PATH>`?

Reply:
No we shouldn't as that would go against the standard we set for the other command examples.

Reply:
In that case could we update the comment to reflect this since it implies it uses `serverdeprovision.yaml` by default?

Reply:
I agree with @andrewbu-ccbill on this. The filename in that example is only being given to match with the file example that we provide.
We are using this standard for all our examples.